docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md

```diff
@@ -28,18 +28,18 @@ Review the reference information before you start, or just look up the necessary
 ## Creating a voucher
 
 To create a discount voucher:
-1. On the *Discount* page,  in the top-right corner, click **Create new discount**.
-2. On the *Create new discount* page, in the *General* tab, do the following:
-    1. In *STORE RELATION*, select the stores where you wish the discount to be active.
-    2. In the *DISCOUNT TYPE* drop-down, select **Voucher codes**.
+1. On the **Discount** page,  in the top-right corner, click **Create new discount**.
+2. On the **Create new discount** page, on the **General Information** tab, do the following:
```

Comment:
```suggestion
2. On the **Create new discount** page, in the **General Information** tab, do the following:
```

Reply:
According to https://developers.google.com/style/ui-elements, the _on_ preposition should be used here.